pylmod/tests/test_gradebook.py

```diff
@@ -267,7 +327,21 @@ def test_get_gradebook_id(self):
         # Remove data and assert exception raised
         self._register_get_gradebook(False)
         with self.assertRaises(PyLmodUnexpectedData):
-            gradebook_id = test_gradebook.get_gradebook_id(self.GBUUID)
+            test_gradebook.get_gradebook_id(self.GBUUID)
+
+    @httpretty.activate
+    def test_get_options(self):
+        """Verify that we can get the options for a gradebook."""
+        self._register_get_options(True)
+        self._register_get_gradebook()
+        gradebook = GradeBook(self.CERT, self.URLBASE, self.GBUUID)
+        options = gradebook.get_options(gradebook_id=self.GRADEBOOK_ID)
+        self.assertIn('membershipQualifier', options)
+
+        # check for no data
+        self._register_get_options(False)
```

Comment:
This one is my bad as discussed IRL. You may want to change it to a function that accepts the body though so that you don't need conditional statements, but it is fine as is